### PR TITLE
Added labels guide

### PR DESCRIPTION
**Pull request by CharlyJazz**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -98,3 +98,9 @@ fine).
 
 [ghi]: http://github.com/sinatra/sinatra/issues
 [ml]: http://groups.google.com/group/sinatrarb/topics "Sinatra Mailing List"
+
+* ["Help Wanted"](https://github.com/sinatra/sinatra/labels/help%20wanted):  Anyone willing to pitch in is open to contribute to this ticket as they see fit (will try to add context / summarize or ask for requirements)
+
+* ["Good First Issue"](https://github.com/sinatra/sinatra/labels/good%20first%20issue): Potential first time contributors should start here
+
+* ["Wishlist"](https://github.com/sinatra/sinatra/labels/Wishlistgit): All the things I wish we had but have no time for
```
Comment (zzak) [inline CONTRIBUTING.md]: It looks like the link for Wishlist is wrong, there is a "git" at the end that should be removed :bow:

Comment (CharlyJazz) [reply CONTRIBUTING.md]: FIxed :sob:

**Comment by zzak** [comment]
@CharlyJazz Sweet, do you mind squashing the commits?

**Comment by CharlyJazz** [comment]
@zzak, I have no idea how to do that.

But I understand what you mean, I've created several unnecessary commits.

**Comment by zzak** [comment]
@CharlyJazz Nevermind, I forgot you can do that in the UI now! :tada:

**Comment by CharlyJazz** [comment]
@zaak What you think about this issue ? https://github.com/sinatra/sinatra.github.com/issues/231

Nobody has commented anything about it. Should something be done?

**Comment by zaak** [comment]
@CharlyJazz I think these broken links should be fixed, definitely.

**Pull request merged**